src/react-apps/applications/shared/__tests__/LeftDrawerMenu.test.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
+import { MemoryRouter } from 'react-router';
+import * as renderer from 'react-test-renderer';
+import LeftDrawerMenu from '../src/navigation/drawer/LeftDrawerMenu';
+//import { createRender } from '@material-ui/core/test-utils';
+
+describe('render left drawer menu', () => {
+  let mockMenuType: string;
+  let mockactiveLeftMenuSelection: string;
+
+  it('renders left drawer menu for create option in header without crashing', () => {
+    mockMenuType = 'create';
+    mockactiveLeftMenuSelection = 'gui';
+
+    const rendered =
+      renderer.create(
+        <MemoryRouter>
+          <LeftDrawerMenu
+            menuType={mockMenuType}
+            activeLeftMenuSelection={mockactiveLeftMenuSelection}
+          />
+        </MemoryRouter>,
+      );
+    expect(rendered).toMatchSnapshot();
+  });
+
+  it('renders left drawer menu for about option in header without crashing', () => {
+    mockMenuType = 'about';
+    mockactiveLeftMenuSelection = 'about';
+
+    const rendered =
+      renderer.create(
+        <MemoryRouter>
+          <LeftDrawerMenu
+            menuType={mockMenuType}
+            activeLeftMenuSelection={mockactiveLeftMenuSelection}
+          />
+        </MemoryRouter>,
+      );
+    expect(rendered).toMatchSnapshot();
+  });
+
+  it('renders left drawer menu for language option in header without crashing', () => {
+    mockMenuType = 'language';
+    mockactiveLeftMenuSelection = 'language';
+
+    const rendered =
+      renderer.create(
+        <MemoryRouter>
+          <LeftDrawerMenu
+            menuType={mockMenuType}
+            activeLeftMenuSelection={mockactiveLeftMenuSelection}
+          />
+        </MemoryRouter>,
+      );
+    expect(rendered).toMatchSnapshot();
+  });
+
+  it('renders left drawer menu for language option in header without crashing', () => {
+    mockMenuType = 'test';
+    mockactiveLeftMenuSelection = 'test';
+
+    const rendered =
+      renderer.create(
+        <MemoryRouter>
+          <LeftDrawerMenu
+            menuType={mockMenuType}
+            activeLeftMenuSelection={mockactiveLeftMenuSelection}
+          />
+        </MemoryRouter>,
+      );
+    expect(rendered).toMatchSnapshot();
+  });
+
+  it('renders left drawer menu for language option in header without crashing', () => {
+    mockMenuType = 'publish';
+    mockactiveLeftMenuSelection = 'publish';
+
+    const rendered =
+      renderer.create(
+        <MemoryRouter>
+          <LeftDrawerMenu
+            menuType={mockMenuType}
+            activeLeftMenuSelection={mockactiveLeftMenuSelection}
+          />
+        </MemoryRouter>,
+      );
+    expect(rendered).toMatchSnapshot();
+  });
+});
+
+describe('returns empty div without crashing', () => {
```

Comment:
This is supposed to be a test-case, so a better text would be 
`when menuType and activeLeftMenuSelection is undefined or null`

Reply:
Fixed